my-app/src/components/login/login.js

```diff
@@ -1,73 +1,74 @@
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/no-unused-state */
+/* eslint-disable no-console */
 import React from 'react';
 import '../../app.css';
-import {Link, withRouter} from "react-router-dom";
-import axios from 'axios';
-
+import { Link, withRouter } from 'react-router-dom';
+// import axios from 'axios';
```

Comment:
if not being used, just delete this import

Reply:
good to know we were not using this. as I kept getting a warning for it not being used